R/toolAggregate.R

```diff
@@ -274,7 +274,7 @@ toolAggregate <- function(x, rel, weight = NULL, from = NULL, to = NULL, dim = 1
 
         tmp <- unique(sub(search, "\\1#|TBR|#\\3", names))
         additions <- strsplit(tmp, split = "#|TBR|#", fixed = TRUE)
-        add <- sapply(additions, function(x) return(x[1:2]))
+        add <- sapply(additions, function(x) return(x[1:2])) #nolint
```

Comment:
```suggestion
        add <- vapply(additions, function(x) return(x[1:2]), character(2))
```

Reply:
to avoid breaking something I would like to stick to sapply here